components/header/_header.scss

```diff
@@ -1,30 +1,65 @@
-$header-bp: 36em;
-
+// Block
 .header {
   display: block;
-  overflow: hidden;
-  padding: 1.4rem 0 1rem;
-  &::before {
-    content: "";
-    display: block;
+  position: relative;
+  margin: 0 0 2rem;
+
+  // Elements
+  &__top {
     width: 100%;
-    height: 0.45rem;
-    background-color: $color-secondary;
-    position: absolute;
-    top: 0;
-    left: 0;
+    padding: .7rem 0;
+    background: $color-secondary;
+  }
+
+  &__logo {
+    height: 1.333rem;
+  }
+
+  &__main {
+    padding: 1.1rem 0;
+    background: $color-header-main;
+  }
+
+  &__nav {
+    background: $color-header-main;
+    @include mq(xs,s) {
+      background: darken($color-header-main, 5%);
+    }
+  }
+
+  &__title {
+    color: $color-white;
+    font-size: 1.556rem;
+    margin: 0;
+  }
```

Comment:
Font weight appears to heavy on Firefox. (There's a mixin for this in another PR)

### Without '-moz-osx-fontsmoothing'

![ons-header-no-moz-antialiasing](https://user-images.githubusercontent.com/22910336/41610414-0e1428ea-73e5-11e8-8751-a13489b2ef66.png)

### With '-moz-osx-fontsmoothing'
![ons-header-with-moz-antialiasing](https://user-images.githubusercontent.com/22910336/41610429-1820d158-73e5-11e8-921b-6240e86675ae.png)


`-moz-osx-font-smoothing: grayscale;` required.